tests/framework/layer/CMakeLists.txt

```diff
@@ -53,6 +54,48 @@ target_link_libraries(test_layer_export_version_2 PRIVATE test_layer_deps)
 target_compile_definitions(test_layer_export_version_2 PRIVATE
     ${TEST_LAYER_VERSION_0_EXPORTS} ${TEST_LAYER_VERSION_1_EXPORTS} ${TEST_LAYER_VERSION_2_EXPORTS} )
 
+# Physical device list altering layer (add phys dev, remove phys dev, re-order phys devs)
+set(PHYSDEV_LAYER_ADD_EXPORTS
+        ${TEST_LAYER_VERSION_0_EXPORTS} ${TEST_LAYER_VERSION_1_EXPORTS} ${TEST_LAYER_VERSION_2_EXPORTS}
+        TEST_LAYER_NAME="VkLayer_LunarG_add_phys_dev"
+        TEST_PHYSDEV_LAYER_ADD=1
+    )
```

Comment:
Is there a particular reason this behavior was set at compile time rather than at runtime? I'm not suggesting it has to be at runtime, but I'm curious as to why.

Reply:
That was the mechanism that was used.  Plus telling it to add remove or resort layers would require some other communication mechanism (like a pnext chain item or a new entry-point that we'd have to export).  If you think of anything cleaner, I'm all ears.